src/reverb/doc/cron-for-ovh/cron-sync-orders.php

```diff
@@ -0,0 +1,16 @@
+<?php
+// Get cURL resource
+$curl = curl_init();
+// Set some options - we are passing in a useragent too here
+curl_setopt_array(
+    $curl,
+    array(
+        CURLOPT_RETURNTRANSFER => 1,
+        CURLOPT_URL => '[ADD YOUR DOMAIN URL]/modules/reverb/cron.php?code=orders',
```

Comment:
Is there text somewhere to tell the seller who is on OVH how to do this? I feel like we're just going to get a lot of support requests.